### PR TITLE
Prepare 1.3.0 release

### PR DESCRIPTION
**Pull request by dholbach**
Also point out how to create the dockerhub.yaml file, update other release related instructions.

closes: #97

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -77,15 +77,30 @@ real life testing.
 
 ## Publishing a new kured release
 
-This is the most straight-forward in the whole process.
+Check that `README.md` has an updated compatibility matrix and that the
+url in the `kubectl` incantation (under "Installation) is updated to the
```
Comment (hiddeco) [inline DEVELOPMENT.md]:
```suggestion
url in the `kubectl` incantation (under "Installation") is updated to the
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -77,15 +77,30 @@ real life testing.
...
+url in the `kubectl` incantation (under "Installation") is updated to the
+new version you want to release.
+
+Now create the `kured-<release>-dockerhub.yaml`:
+
+```sh
+VERSION=1.3.0
+MANIFEST="kured-$VERSION-dockerhub.yaml"
+cat kured-rbac.yaml > "$MANIFEST"
+echo "---" >> "$MANIFEST"
```
Comment (hiddeco) [inline DEVELOPMENT.md]: Not sure this is actually required, following this resulted in a:

```yaml
<snip>
apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
  namespace: kube-system
  name: kured
subjects:
- kind: ServiceAccount
  namespace: kube-system
  name: kured
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: Role
  name: kured
---
---
apiVersion: v1
kind: ServiceAccount
metadata:
  name: kured
  namespace: kube-system
<snip>
```

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -77,15 +77,30 @@ real life testing.
...
+cat kured-ds.yaml >> "$MANIFEST"
+```
+
+The last thing you need to do is update the `image:` to point to the release
```
Comment (hiddeco) [inline DEVELOPMENT.md]: Can be simply done with `sed -i`, but would require the 'no latest tag' command to be moved to e.g. above the `image` key.

```sh
sed "s#docker.io/weaveworks/kured#docker.io/weaveworks/kured:$VERSION#g" "kured-$VERSION-dockerhub.yaml"
```

File: DEVELOPMENT.md (hunk 2 above)
Comment (dholbach) [reply DEVELOPMENT.md]: Hah, yes, you're right - fixing!

File: DEVELOPMENT.md (hunk 3 above)
Comment (dholbach) [reply DEVELOPMENT.md]: Nice one!

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -77,15 +77,30 @@ real life testing.
...
+new version you want to release.
+
+Now create the `kured-<release>-dockerhub.yaml` for e.g. `1.3.0`:
+
+```sh
+VERSION=1.3.0
+MANIFEST="kured-$VERSION-dockerhub.yaml"
+cat kured-rbac.yaml > "$MANIFEST"
+cat kured-ds.yaml >> "$MANIFEST"
+sed -i "s#docker.io/weaveworks/kured#docker.io/weaveworks/kured:$VERSION#g" "kured-$VERSION-dockerhub.yaml"
```
Comment (hiddeco) [inline DEVELOPMENT.md]: You used the old(er) cached version of my suggestion, using `$MANIFEST` here is probably what you want.

**Comment by dholbach** [comment]
Thanks @hiddeco - yes, one step at a time. :checkered_flag:

**Pull request merged**